src/CodeFormatter/Program.cs

```diff
@@ -159,18 +168,32 @@ private static int Main(string[] args)
             engine.ConvertUnicodeCharacters = convertUnicode;
             engine.Verbose = verbose;
 
-            Console.WriteLine(Path.GetFileName(projectOrSolutionPath));
-            string extension = Path.GetExtension(projectOrSolutionPath);
-            if (StringComparer.OrdinalIgnoreCase.Equals(extension, ".sln"))
+            Console.WriteLine(Path.GetFileName(projectSolutionOrRspPath));
+            string extension = Path.GetExtension(projectSolutionOrRspPath);
+            if (StringComparer.OrdinalIgnoreCase.Equals(extension, ".rsp"))
+            {
+                using (var workspace = ResponseFileWorkspace.Create())
+                {
```

Comment:
Should add the following line:

``` csharp
workspace.LoadMetadataForReferencedProjects = true;
```

This can help increase performance by loading referenced projects as metedata vs. source.  Shows up mostly with larger solutions (aka Roslyn).